client/main.js

```diff
@@ -15,35 +15,39 @@ function Event({ event }) {
   return <div className={ `event list-group-item ${ eventCategory ? eventCategory.toLowerCase() : null }` }>
     <h2>{ event.title } <EventTime time={event.time} /> </h2>
     <p>
-      <span>{ event.info && event.info.Lokacija ? event.info.Lokacija : null }</span>
-      <span> | </span>
-      <span>{ eventCategory }</span>
+      <h4>{ event.info && event.info.Lokacija ? event.info.Lokacija : null }</h4>
+      <span className="label label-primary">{ eventCategory }</span>
     </p>
     <p dangerouslySetInnerHTML={ eventDescription(event.description) }></p>
     <EventLink link={ event.link } />
   </div>;
 }
 
-function Date({ date }) {
-  const dateEvents = date.events.map(event => <Event event={event} />)
+function Date({ date, events }) {
+  const eventList = events.map(event => <Event event={event} />);
+  const dateChunks = date.split('/');
+  const normalizedDateString = `${dateChunks[1]}/${dateChunks[0]}`
+  const formatedDate = format(normalizedDateString, 'D. MMMM', { locale: hrLocale })
```

Comment:
Migrate initial date parsing into server part and make format as property of this component with default value you used for formatting :necktie: